NLog.Web.Tests/LayoutRenderers/AspNetSessionValueLayoutRendererTests.cs

```diff
@@ -23,6 +24,7 @@ public AspNetSessionValueLayoutRendererTests()
 
         public void SetUp()
         {
+            ConfigurationItemFactory.Default.LayoutRenderers.RegisterDefinition("aspnet-session", typeof(AspNetSessionValueLayoutRenderer));
```

Comment:
@petemounce please add the following code to the erroneous test case

```
 InternalLogger.LogFile = @"C:\Temp\nlog.web.log";
 InternalLogger.LogLevel = LogLevel.Trace;
```

and post the log :)